src/layouts/BlogPostLayout.astro

```diff
@@ -42,6 +42,18 @@ import "../styles/global.css";
           <h1 class="text-3xl">{frontmatter.title}</h1>
         </header>
         <slot />
+        <style is:global>
```

Comment:
Do we have to use global here? From my understanding the default of scoped styling should be enough for the blog posts.

Otherwise I would check if we can use tailwind here instead to keep using the same styling method.

Reply:
oh yeah we should probably not use that. Just copy pasted from somewhere

Reply:
ah okay. Good solution to put it under the blog class